src/MicrosoftTeams.schema.json

```diff
@@ -129,56 +173,31 @@
                         "description": "A unique identifier for the bot that matches its ID in the bot framework.",
                         "maxLength": 64
                     },
-                    "pinnedTabs": {
+                    "scope": {
                         "type": "array",
                         "items": {
-                            "type": "object",
-                            "properties": {
-                                "id": {
-                                    "type": "string",
-                                    "description": "A unique identifier for this tab instance.",
-                                    "maxLength": 64
-                                },
-                                "definitionId": {
-                                    "type": "string",
-                                    "description": "The id of the tab definition this tab is an instance of.",
-                                    "maxLength": 64
-                                },
-                                "displayName": {
-                                    "type": "string",
-                                    "description": "The display name of this tab instance.",
-                                    "maxLength": 16
-                                },
-                                "url": {
-                                    "type": "string",
-                                    "description": "The content url for this tab instance"
-                                },
-                                "websiteUrl": {
-                                    "type": "string",
-                                    "description": "The website url of the content displayed in this tab instance"
-                                }
-                            },
-                            "required": [
-                                "id",
-                                "definitionId",
-                                "displayName",
-                                "url"
-                            ]
+                            "enum": [
+                                "team",
+                                "people",
+                                "personal"
+                            ],
+                        "description": "Specifies whether the bot offers an experience in the context of a channel in a team, a chat between one or more people, or an experience scoped to an individual user alone. These options are non-exclusive"
                         }
                     }
                 },
                 "required": [
-                    "mri"
+                    "mri",
```

Comment:
Sujeet suggested that we move away from using "mri" in the schema since that's an internal acronym we use for any contact identifier. We should just call this "id" or "botId".